app/src/main/java/org/mozilla/fenix/share/ShareFragment.kt

```diff
@@ -111,6 +116,20 @@ class ShareFragment : AppCompatDialogFragment() {
         }
         shareToAppsView = ShareToAppsView(binding.appsShareLayout, shareInteractor)
 
+        if (FeatureFlags.saveToPDF) {
+            binding.dividerLineAppsShareAndPdfSection.isVisible = true
+            binding.savePdf.apply {
+                isVisible = true
+                setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+                setContent {
+                    FirefoxTheme {
+                        SaveToPDFItem {
+                            shareInteractor.onSaveToPDF(tabId = args.sessionId)
```

Comment:
nit: maybe it's just me, but it looks strange having the lambda out of the parenthesis, in a "tree" of composables content. 😅 . 
Maybe you could leave it on the same row as `SaveToPDFItem` so it looks more like a method call rather than an "inside" composable.